lib/spack/spack/config.py

```diff
@@ -763,6 +768,22 @@ def _add_platform_scope(
     cfg.push_scope(scope_type(plat_name, plat_path))
 
 
+def _add_configuration_paths_from_entry_points(configuration_paths):
```

Comment:
(minor) Here I would just define:
```suggestion
def paths_from_entry_points() -> List[str]:
```
and return the list of paths from entry points